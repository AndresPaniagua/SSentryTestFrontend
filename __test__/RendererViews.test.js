import React, { Fragment, useEffect } from "react";
import { useHistory } from "react-router-dom";
import ReactDOM from "react-dom";
import Enzyme, { shallow, render, mount } from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import Company from "../src/views/Company";
import Home from "../src/views/Home";

Enzyme.configure({ adapter: new Adapter() });

describe("Renderer Views", () => {
  describe("when component call is successful", () => {
    test("Company should correctly", async () => {
      // Arrange
      const tag = <Company/>;

      // Act
      const wrapper = shallow(tag).dive();

      // Assert
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe("when component call is successful", () => {
    test("Home should correctly", async () => {
      // Arrange
      const tag = <Home/>;

      // Act
      const wrapper = shallow(tag).dive();

      // Assert
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
