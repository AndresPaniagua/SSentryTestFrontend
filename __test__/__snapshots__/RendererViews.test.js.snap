// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderer Views when component call is successful Company should correctly 1`] = `
<div
  className="home-view container"
>
  <RegisterForm />
</div>
`;

exports[`Renderer Views when component call is successful Home should correctly 1`] = `
<div
  className="home-view container"
>
  <MainForm />
</div>
`;
